### ISSUE TITLE
Update sidebar link styling, update link decoration

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa801#7728]
# Problem

Sidebar links default as plain dark text and are hard to distinguish from other text in the sidebar.

# Background

              >     * Wasn't there EoD talk about making the links in the sidebar look like links? There is no underline, color change, or hover effect.

The default behavior of the card links are black due to the how the card handles background colors: https://sandbox.prod.drupal.uiowa.edu/people/sean-adams-hiett.    We can either process in the `bg--none` class like on https://github.com/uiowa/uiowa/blob/main/docroot/sites/its.uiowa.edu/modules/its_core/its_core.module#L522 or shift our default for this layout to `bg--none` but that would require some greater refactoring and testing.

_Originally posted by @bspeare in https://github.com/uiowa/uiowa/issues/7695#issuecomment-2135698232_

# Proposed solution

- [ ] Shift our default for the layout to `bg--none` and check against existing splits for a consistent, easy to interpret interface across the platform.
- [ ] Remove one-off adjustments.

**Comment by joewhitsitt** [comment]
This design pattern also surfaces on the contact block.

https://intradent.dentistry.uiowa.edu/tms
